main.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-unused-vars, no-use-before-define */
+
+const bookArray = localStorage.getItem('books') ? JSON.parse(localStorage.getItem('books')) : [];
+localStorage.setItem('books', JSON.stringify(bookArray));
+const library = JSON.parse(localStorage.getItem('books'));
+
+function Book(title, author) {
+  this.title = title;
+  this.author = author;
+}
+
```

Comment:
- [ ] Kindly ensure `arrow functions` are used instead of the `function keyword` if you check the Javascript requirements you would see that the use of ES6 is encouraged. Javascript's ES6 brings new syntax and awesome features to make your code more modern and readable. It allows you to write less code and do more. ES6 introduces us to many great features like arrow functions. The arrow function is really awesome and makes your code more readable, more structured, and look modern. Take a minute and also go through this article on why[ ES6 arrow functions are so awesome]( https://zendev.com/2018/10/01/javascript-arrow-functions-how-why-when.html). 😄